core-bundle/src/Routing/Page/PageRoute.php

```diff
@@ -20,25 +20,14 @@
 
 class PageRoute extends Route implements RouteObjectInterface
 {
-    /**
-     * @var PageModel
-     */
-    private $pageModel;
-
-    /**
-     * @var string
-     */
-    private $urlPrefix;
-
-    /**
-     * @var string
-     */
-    private $urlSuffix;
+    private PageModel $pageModel;
+    private ?string $urlPrefix;
+    private ?string $urlSuffix;
 
     /**
      * The referenced content object.
      */
-    private $content;
+    private ?object $content = null;
```

Comment:
@aschempp Is this correct? Is `$content` either an object or `null`?

Reply:
As briefly discussed, `$content` can be anything and can therefore not be type hinted.